internal/plugins/ansible/v1/scaffolds/internal/templates/requirements.go

```diff
@@ -34,6 +34,6 @@ func (f *RequirementsYml) SetTemplateDefaults() error {
 const requirementsYmlTmpl = `---
 collections:
   - name: community.kubernetes
-    version: "<1.0.0"
+    version: "<1.1.0"
```

Comment:
Perhaps we can do this so the latest patch release is used. I'm not sure what compatibility looks like for minor versions, so this is a reasonable compromise.
```suggestion
    version: "~=1.1"
```

Reply:
im just testing stuff. i'm expecting stuff to explode from <1.0.0 to 1.0.0

Reply:
but yes in the final version i think i'll do something like that

Reply:
Supposedly the [`~=` operator](https://www.python.org/dev/peps/pep-0440/#compatible-release) should work, but I guess not.